.github/workflows/add_validate_build_all.sh

```diff
@@ -0,0 +1,45 @@
+#!/usr/bin/env
+# This is just a shell script copy of add_validate_build_all.yaml
```

Comment:
this is "dead" code that will be hard to remember to maintain. I would suggest just removing all of this from the yaml and call this script instead.